data/shaders/basic_vs.glsl

```diff
@@ -1,13 +1,22 @@
 #version 150 core
-#include <locals>
 #include <globals>
 
 in vec4 a_Position;
 in vec4 a_Normal;
 in vec2 a_TexCoord;
 out vec2 v_TexCoord;
+out vec4 v_Color;
+
+in vec4 i_World0;
```

Comment:
would it make sense to move those instance inputs into a header ("locals")? hint: it will not if different shaders have different instance inputs

Reply:
Well, most of the shaders use it, so I'll move it in header.

Reply:
The instance variables are likely to change, so I'd say not.